src/components/Gallery/Gallery.tsx

```diff
@@ -0,0 +1,22 @@
+import './Gallery.scss'
+
+type GalleryProps = { galleryArray: string[] }
+
+const Gallery = ( { galleryArray }: GalleryProps) => {
+
+const gallery = galleryArray ?? ["Sorry! Image not found."]
+
+  return (
+
+    <div className="gallery"> 
+      {gallery.map((image: string) => {
```

Comment:
Each of these will need a key, you can use the index just pre-pend it with something. 

e.g `gallery-image-${index}`